tardis/visualization/widgets/tests/test_custom_abundance.py

```diff
@@ -88,6 +89,10 @@ def caw(yml_data):
         CustomAbundanceWidget generated from a YAML
     """
     caw = CustomAbundanceWidget(yml_data)
+    monkeysession.setattr(
+        "tardis.visualization.widgets.custom_abundance.is_notebook",
+        lambda: True,
+    )
```

Comment:
Was the issue for Convergence plots or for the custom abundance widget?
Also, I think we should raise an error but I also would like to hear what others say about this. If we do so, maybe then we would want a test that mimics the command line and see if it raises an error?

Reply:
> Was the issue for Convergence plots or for the custom abundance widget?

You're right. I just noticed that the issue was created for the convergence plots. But the code was also visible for the three Widgets, so I made the changes. 

I will add the checks for command-line in the Convergence plots.

> Also, I think we should raise an error but I also would like to hear what others say about this. If we do so, maybe then we would want a test that mimics the command line and see if it raises an error?

Raising an error would definitely be a more prudent way to handle it.

Reply:
@atharva-2001 I have added a check where RuntimeError is raised if `run_tardis` is called from the command line and if `show_convergence_plots` is set to True (which was the only case where the code was being printed). 

The default value of `show_convergence_plots` was set to True in `simulation/base.py` but False in `tardis/base.py`. I changed the former to False as well, otherwise the tests which simply call `Simulation.from_config` using the default as True would've failed.